_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -156,3 +191,49 @@ Option | Required | Type | Description
 `schema` | Yes | String | The Avro [schema declaration](https://avro.apache.org/docs/current/specification/#schema-declaration). Not required if `auto_schema` is set to true.
 `auto_schema` | No | Boolean | When set to `true`, automatically generates the Avro [schema declaration](https://avro.apache.org/docs/current/specification/#schema-declaration) from the first event.
 
+### Setting a schema with Parquet
+
+The following example shows how you can configure the `s3` sink to write Parquet data with a schema for the AWS VPC Flow logs.
```

Comment:
The meaning after "sink" is a little unclear. Should we say to where the data is being written? Also clarify what we mean by "with a schema for". I assumed that we are referring to VPC Flow Logs, the Amazon VPC feature.

Reply:
@natebower , How about this?

> The `s3` sink can write data in Parquet format using a schema that you specify. The following example shows how you can do this. The example creates a schema based on the AWS VPC Flow logs data model.

Yes, this is the Amazon VPC feature. It is documented here: https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html#flow-log-records